jina/peapods/runtimes/zmq/zed.py

```diff
@@ -71,7 +71,10 @@ def _load_executor(self):
         """Load the executor to this runtime, specified by ``uses`` CLI argument."""
         try:
             self._executor = BaseExecutor.load_config(
-                self.args.uses, pea_id=self.args.pea_id, read_only=self.args.read_only
+                self.args.uses,
+                pea_id=self.args.pea_id,
+                replica_id=getattr(self.args, 'replica_id', -1),
```

Comment:
If I interpret this right, then being inside the runtime of a Pod with one replica is not the same as being inside the runtime of the first replica inside a Pod with two replicas. I am not against it, but: Why? If I remember correctly, we had the same pattern for shards and then switched to having one shard is handled in the same way as having multiple.

(We don't need to change this now, I just wanted to know, whether this is by accident or by consciously choosing it.)

Reply:
If u have one Replica u use Pod and not CompoundPod.

Reply:
is that what u mean?

Reply:
We had a short call. The question was if we should have the same workspace pattern even if replicas are 1.
This means storing the index files in:
```
'vecidx-0-0'
'vecidx-0-1'
'vecidx-0-2'
```
instead of 
```
'vecidx-0'
'vecidx-1'
'vecidx-2'
```
This is on purpose in to not break existing Jina implementations.

Reply:
I would not break. And since this will suffer quite some changes, I would be conservative

Reply:
at least until this is more mature